src/OpenSage.Game/Gui/DrawingContext2D.cs

```diff
@@ -109,14 +109,7 @@ private void UpdateOpacity()
             in Rectangle destinationRect,
             in bool flipped = false)
         {
-            var color = ColorRgbaF.White.WithA(_currentOpacity);
-
-            _spriteBatch.DrawImage(
-                texture,
-                sourceRect,
-                RectangleF.Transform(destinationRect.ToRectangleF(), _currentTransform),
-                color,
-                flipped);
+            DrawImage(texture, sourceRect, destinationRect.ToRectangleF(), flipped);
```

Comment:
Why you removed the opacity here?

Reply:
Because the overloaded function it calls calculates the opacity already